components/apimgt/org.wso2.carbon.apimgt.rest.api.commons/src/main/java/org/wso2/carbon/apimgt/rest/api/common/interceptors/RESTAPISecurityInterceptor.java

```diff
@@ -105,6 +105,7 @@ public boolean preCall(Request request, Response response, ServiceMethodInfo ser
                 return false;
             }
         } else if (requestURI.contains("/store")) {
+            /*TODO: Remove below swagger.json serving path when old Store UUF app is removed ~tmkb*/
```

Comment:
This has been fixed by this [commit](https://github.com/wso2/carbon-apimgt/pull/4357/commits/4f2e377f87f6826849b402dcd0595f94edfc6f9f) , Please review